op-node/cmd/genesis/cmd.go

```diff
@@ -52,6 +52,10 @@ var Subcommands = cli.Commands{
 				return err
 			}
 
+			if err := config.Check(); err != nil {
```

Comment:
Is this check duplicated on line 145 below?

Reply:
That is a different command